style.css

```diff
@@ -2,82 +2,202 @@
 
 body {
   box-sizing: border-box;
-  max-width: 375px;
+  max-width: 100%;
   background-color: white;
   margin: 0;
+  font-family: 'Inter', sans-serif;
+  margin-left: 32px;
+}
+
+* {
+  margin: 0;
+}
+
+.menu-icon,
+.welcome {
+  display: none;
 }
 
 nav ul {
   display: flex;
-  justify-content: space-between;
   list-style-type: none;
-  margin-right: 36px;
-  margin-top: 54px;
-  height: 28px;
+  width: 20%;
+  margin: 26px 40% 0 40%;
 }
 
-nav li a.welcome {
+nav a {
   text-decoration: none;
-  font-weight: 600;
+  margin-right: 12px;
+  font-size: 15px;
+  color: #344563;
+  padding: 10px 12px;
   font-family: 'Inter', sans-serif;
-  color: #28352f;
-  font-size: 20px;
-  line-height: 28px;
-  margin: 0;
-  width: 90px;
+  font-weight: 600;
 }
 
-section.myname {
-  margin: 0;
-  background: url(img/bgimage.png);
-  background-repeat: no-repeat;
-  background-size: cover;
+@media screen and (max-width: 375px) {
```

Comment:
- [ ] Kindly use mobile first approach as stated in the project requirements, which means the standard design should be for mobile view, then you add media queries for desktop view `min-width: 768px` 👍

Reply:
Please, can you kindly show me where it was stated as project requirement that we must use mobile first approach ?

Reply:
What do you have to say about software developers that combine max-width and min-width?

is it not a good practice?

Reply:
Do mobile-first approach advantages outweigh its drawbacks?